test/api/unit_roles_test.rb

```diff
@@ -9,14 +9,24 @@ def app
     Rails.application
   end
 
-  # GET /api/units_roles
+  # Get unit_roles
   def test_get_unit_roles
-    get '/api/unit_roles'
+    get with_auth_token'/api/unit_roles'
+	  assert_equal 200, last_response.status
     # UnitRole.joins(:role, :unit).where("user_id = :user_id and roles.name <> 'Student'", user_id: user.id)
 
     # asserts
   end
 
+  # Get unit_roles_details
+  def test_get_unit_roles_details
+    ur = UnitRole.second
+    id_of_ur = ur.id
+
+    get with_auth_token "/api/unit_roles/#{ur.id}"
+    assert_equal 200, last_response.status
```

Comment:
Same here... just returning 200 doesn't mean anything other than the request got through and returned.